app/com/arpnetworking/play/configuration/ConfigurationHelper.java

```diff
@@ -111,6 +113,17 @@ public static FiniteDuration getFiniteDuration(final Config configuration, final
                 .getInstance(clazz);
     }
 
+    /**
+     * Resolves and converts a piece of a {@link Config} to a JSON string.
+     *
+     * @param config The config to be rendered as JSON.
+     * @param path The path to the sub-object to render as JSON.
+     * @return The JSON representation of that sub-object.
+     */
+    public static String toJson(final Config config, final String path) {
+        return config.withFallback(ConfigFactory.systemProperties()).resolve().getObject(path).render(ConfigRenderOptions.concise());
```

Comment:
I'm concerned about your fallback here.  All of the fallbacks should already be applied when the config was initialized.

Reply:
Ah, thanks! I can fix that -- I just cargo-culted wrong from MAD.